tests/core/pyspec/eth2spec/test/context.py

```diff
@@ -361,6 +363,38 @@ def wrapper(*args, spec: Spec, **kw):
     return decorator
 
 
+def with_config_overrides(config_overrides):
```

Comment:
Added this function for testing, but refrained from using it in a test, I'm not happy with it yet. Swapping configuration and reloading specs per test case is not pretty, at all. If the spec was an object and the config was a property, this would be much cleaner, but it may hurt readability in the markdown docs.

Reply:
This is resolved, no module reloading necessary anymore, function has been updated.